File: takasugi.pl
#!/usr/bin perl
# perl takasugi.pl
use strict;
use warnings;

foreach(1..100) {
    if    ($_ % 15 == 0) {print "FizzBuzz\n";}
    elsif ($_ % 3  == 0) {print "Fizz\n";}
    elsif ($_ % 5  == 0) {print "Buzz\n";}
    else                 {print $_ , "\n"; }
};

# これでもいける
# map {
# ...
# }(1..100);

File: takasugi.js
// node takasugi.js

'use strict';

for (var i = 1; i <= 100; i++) {
  var echo_text = '';
  if (i % 3 == 0) {
    echo_text += 'Fizz';
  }
  if (i % 5 == 0) {
    echo_text += 'Buzz';
  }
  if (echo_text === '') {
    echo_text = i;
  }
  console.log(echo_text);
}
